browser/ui/views/omnibox/brave_omnibox_view_views.cc

```diff
@@ -27,3 +36,58 @@ bool BraveOmniboxViewViews::SelectedTextIsURL() {
                              &write_url);
   return write_url;
 }
+
+void BraveOmniboxViewViews::UpdateContextMenu(
+    ui::SimpleMenuModel* menu_contents) {
+  OmniboxViewViews::UpdateContextMenu(menu_contents);
+  if (SelectedTextIsURL()) {
+    BraveUpdateContextMenu(menu_contents,
+                           controller()->GetLocationBarModel()->GetURL());
+  }
+}
+
+#if BUILDFLAG(IS_WIN)
+bool BraveOmniboxViewViews::AcceleratorPressed(
+    const ui::Accelerator& accelerator) {
+  ui::KeyEvent event(
+      accelerator.key_state() == ui::Accelerator::KeyState::PRESSED
+          ? ui::ET_KEY_PRESSED
+          : ui::ET_KEY_RELEASED,
+      accelerator.key_code(), accelerator.modifiers());
+  auto command = GetCommandForKeyEvent(event);
+
+  if ((GetTextInputType() != ui::TEXT_INPUT_TYPE_PASSWORD) &&
+      (command != ui::TextEditCommand::COPY || !SelectedTextIsURL())) {
+    return OmniboxViewViews::AcceleratorPressed(accelerator);
+  }
+  ExecuteCommand(IDC_COPY_CLEAN_LINK, 0);
+  return true;
+}
+
+bool BraveOmniboxViewViews::GetAcceleratorForCommandId(
+    int command_id,
+    ui::Accelerator* accelerator) const {
+  bool is_url = const_cast<BraveOmniboxViewViews*>(this)->SelectedTextIsURL();
+  if (is_url) {
+    if (command_id == kCopy) {
+      return false;
+    }
+    if (command_id == IDC_COPY_CLEAN_LINK) {
+      *accelerator = ui::Accelerator(ui::VKEY_C, ui::EF_PLATFORM_ACCELERATOR);
+      return true;
+    }
+  }
+  return OmniboxViewViews::GetAcceleratorForCommandId(command_id, accelerator);
+}
+#endif  // BUILDFLAG(IS_WIN)
+
+#if BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_MAC)
+void BraveOmniboxViewViews::ExecuteTextEditCommand(
```

Comment:
pls order these similar to their location in header.

Reply:
done